pytest-embedded-wokwi/pytest_embedded_wokwi/wokwi_cli.py

```diff
@@ -51,15 +55,38 @@ def __init__(
         self.app = app
         self.firmware_resolver = firmware_resolver
 
+        # first need to check if wokwi-cli exists in PATH
+        if shutil.which('wokwi-cli') is None:
+            print('Please install wokwi-cli, by running: curl -L https://wokwi.com/ci/install.sh | sh')
+            # self.install_wokwi() not implemented yet
+
+        child = pexpect.spawn('wokwi-cli --help')
+        try:
+            child.expect(r'Wokwi CLI v(\d+\.\d+\.\d+)', timeout=1)
+            wokwi_cli_version = child.match.group(1).decode('utf-8')
+        except pexpect.TIMEOUT:
+            raise RuntimeError('Failed to run wokwi-cli')
+
+        if Version(wokwi_cli_version) < Version(WOKWI_CLI_MINIMUM_VERSION):
+            raise ValueError(
+                f'Wokwi CLI version {wokwi_cli_version} is not supported. '
+                f'Minimum version required: {WOKWI_CLI_MINIMUM_VERSION}. '
+                f'To update Wokwi CLI run: curl -L https://wokwi.com/ci/install.sh | sh'
+            )
```

Comment:
```suggestion
        try:
            child.expect(r'Wokwi CLI v(\d+\.\d+\.\d+)', timeout=1)
            wokwi_cli_version = child.match.group(1).decode('utf-8')
        except pexpect.TIMEOUT:
            logging.warning('Failed to get wokwi-cli version, assume version requirements satisfied')
        else:
            if Version(wokwi_cli_version) < Version(WOKWI_CLI_MINIMUM_VERSION):
                raise ValueError(
                    f'Wokwi CLI version {wokwi_cli_version} is not supported. '
                    f'Minimum version required: {WOKWI_CLI_MINIMUM_VERSION}. '
                    f'To update Wokwi CLI run: curl -L https://wokwi.com/ci/install.sh | sh'
                )
```

by this approach, if `wokwi-cli --help` changes format one day, at least we don't block the users.